transactions/set-protocol-state-version/2025/apr-3-v2/README.md

```diff
@@ -0,0 +1,17 @@
+# Upgrade Protocol State
+
+This is Step 2 in the Apr 3 2025 Protocol State Upgrade Process. Before doing this step:
+- Must have upgraded NodeVersionBeacon to v1.6.0
+
+[set protocol state version](../../../../transactions/set-protocol-state-version)
+
+To produce arguments:
+```
+jq -n --arg version "2" --arg view "TODO" '[{"type":"UInt64","value":$version},{"type":"UInt64","value":$view}]'
+```
+
+We will definitely be using version 2, but will need to determine the view on the day.
```

Comment:
If I understand correctly we need to wait for the view to pass before proceeding to step 3. We might want to mention that here.